### PR TITLE
new(mp) : New Cisco Callmanager SXML Monitoring Procedure [EN]

### PR DESCRIPTION
**Pull request by psamecentreon**

### CODE REVIEW

File: en/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+---
+id: network-cisco-callmanager-sxml
+title: Cisco Callmanager SXML
+---
+
+## Overview
+
+Using SXML, this Cisco Call Manager Plugin-Pack monitors alerts from the Cisco Unified Communications component.
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md:8]: componentS

File: en/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+```
+
+The command above gets the alerts of a Cisco Callmanager using SXML (```--mode=alerts```).
+It uses api-username (```--api-username='centreoncum'```), an api-password (```--api-password='myapipassword'```)
+and it connects to the Host _mycucm.com_ (```--Hostname='mycucm.com'```) 
+on the port 443 (```--port='443'```) using https (```--proto='https'```).
+By default, the backend method is _curl_ (```--http-backend=curl``).
+
+```bash
+/usr/lib/centreon/plugins//centreon_cisco_cucm_sxml.pl --plugin=network::cisco::callmanager::sxml::plugin \
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md:114]: One option / line

File: en/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+alert [name: CDRFileDeliveryFailureContinues] [severity: critical] [date: Tue Oct  6 05:42:12 2020]:  BillingServerAddress : 172.28.172.105 
+AppID : Cisco CDR Repository Manager ClusterID :  NodeID : server.centreon.com  TimeStamp : Tue Oct 06 05:41:50 EDT 2020. 
+The alarm is generated on Tue Oct 06 05:41:50 EDT 2020.	
+```
+
+The command above gets the alerts of a Cisco Callmanager using SXML (```--mode=alerts```).
+It uses api-username (```--api-username='centreoncum'```), an api-password (```--api-password='myapipassword'```)
+and it connects to the Host _mycucm.com_ (```--Hostname='mycucm.com'```) 
+on the port 443 (```--port='443'```) using https (```--proto='https'```).
+By default, the backend method is _curl_ (```--http-backend=curl``).
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md:111]: Missing `

**Review by Sims24 — CHANGES_REQUESTED** [review]

File: en/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md (hunk 1 above)
Comment (psamecentreon) [reply en/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md]: Fixed ^^

File: en/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md (hunk 2 above)
Comment (psamecentreon) [reply en/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md]: Done

File: en/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md (hunk 3 above)
Comment (psamecentreon) [reply en/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md]: ` added too

**Review by Sims24 — APPROVED** [review]

**Pull request merged**